src/main/markdown/eco/application.md

```diff
@@ -0,0 +1,34 @@
+# Application Frameworks 
+
+One of the main tasks, when writing a GWT application is separating the code and managing the state. GWT offers with
+[MVP Activities and Places](/doc/latest/DevGuideMvpActivitiesAndPlaces.html) modules to do that. Over the
+time other implementations evolved. 
+
+## domino-mvp and domino-historie<a id="domino-mvp-historyt"></a>
```

Comment:
Wrong spelling of domino-history -historie- and the id had an extra 't' at the end.